src/lib.rs

```diff
@@ -410,6 +410,8 @@ mod xproto {
 ///     pub(crate) struct Atoms {
 ///         pub wm_protocols    => b"WM_PROTOCOLS",
 ///         pub wm_del_window   => b"WM_DELETE_WINDOW",
+///         /// Supported EWMH hints
+///         pub _net_supported  => b"_NET_SUPPORTED",
```

Comment:
You can remove the leading `_` on the field name. There's no reason to have the same spelling as the X identifier.